### ISSUE TITLE
haskell-language-server 1.6.1.0 or earlier for GHC 9.2.2 is not available on Linux.

### ISSUE THREAD

**Issue by peacememories** [haskell/vscode-haskell#572]
### Your environment

Which OS do you use:

Fedora Silverblue R35, with VSCode installed in flatpak, using Haskell 9.2.2 via org.freedesktop.Sdk.Extension.haskell

### Steps to reproduce

Install VSCode and haskell like mentioned above and try to open a haskell project.

### Expected behaviour

haskell language server and the vscode extension should work with ghc 9.2.2

### Actual behaviour

The message in the title will appear as an error message in the bottom right and the lsp integration will not work

### Include debug information

Execute in the root of your project the command `haskell-language-server-wrapper --debug .` and paste the logs here (you can find the executable location [here](https://github.com/haskell/vscode-haskell#downloaded-binaries)):

<details>
<summary>
Debug output:
</summary>

```
No 'hie.yaml' found. Try to discover the project type!
Run entered for haskell-language-server-wrapper(haskell-language-server-wrapper-1.6.1.0-linux) Version 1.6.1.0, Git revision f4022c5bb8530cd306c53b941878244bf27a5d41 (dirty) x86_64 ghc-8.10.7
Current directory: <home>/Documents/Projects/highscore-server
Operating system: linux
Arguments: ["--debug"]
Cradle directory: <home>/Documents/Projects/highscore-server
Cradle type: Cabal

Tool versions found on the $PATH
cabal:          3.6.2.0
stack:          Not found
ghc:            9.2.2


Consulting the cradle to get project GHC version...
Project GHC version: 9.2.2
haskell-language-server exe candidates: ["haskell-language-server-9.2.2","haskell-language-server"]
Cannot find any haskell-language-server exe, looked for: haskell-language-server-9.2.2, haskell-language-server
```

</details>

Paste the contents of extension specific log, you can check instructions about how to find it [here](https://github.com/haskell/vscode-haskell#troubleshooting)

<details>
<summary>
Extension log:
</summary>

```
2022-04-08 18:28:15.9890000 [client] INFO Searching for server executables haskell-language-server-wrapper,haskell-language-server in $PATH
2022-04-08 18:28:15.9890000 [client] INFO $PATH environment variable: /app/bin:/app/bin:/app/bin:/usr/bin:/app/tools/docker/bin:/usr/lib/sdk/node16/bin:/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/haskell/bin:<home>/.var/app/com.visualstudio.code/data/node_modules/bin
2022-04-08 18:28:16.0030000 [client] INFO Downloading haskell-language-server
2022-04-08 18:28:16.0040000 [client] INFO Using <home>/.var/app/com.visualstudio.code/config/Code/User/globalStorage/haskell.haskell to store downloaded binaries
2022-04-08 18:28:16.0050000 [client] INFO Fetching the latest release from GitHub or from cache
2022-04-08 18:28:16.9830000 [client] INFO The latest known release is 1.6.1.0
2022-04-08 18:28:16.9830000 [client] INFO Figure out the ghc version to use or advertise an installation link for missing components
2022-04-08 18:28:16.9830000 [client] INFO Working out the project GHC version. This might take a while...
2022-04-08 18:28:16.9970000 [client] INFO Executing '<home>/.var/app/com.visualstudio.code/config/Code/User/globalStorage/haskell.haskell/haskell-language-server-wrapper-1.6.1.0-linux --project-ghc-version' in cwd '/var/home/pickl/Documents/Projects/highscore-server' to get the project or file ghc version
2022-04-08 18:28:17.0750000 [client] INFO Execution of '<home>/.var/app/com.visualstudio.code/config/Code/User/globalStorage/haskell.haskell/haskell-language-server-wrapper-1.6.1.0-linux --project-ghc-version' terminated with code 0
2022-04-08 18:28:17.0760000 [client] INFO The GHC version for the project or file: 9.2.2
2022-04-08 18:28:17.0760000 [client] INFO Search for binary haskell-language-server-Linux-9.2.2 in release assets
2022-04-08 18:28:17.0770000 [client] ERROR haskell-language-server 1.6.1.0 or earlier for GHC 9.2.2 is not available on Linux. [See the list of supported versions here](https://github.com/haskell/vscode-haskell#supported-ghc-versions)
```

</details>

**Comment by fendor** [comment]
Hi, thank you for your bug report! Unfortunately, HLS is not yet available for GHC 9.2.2.

See the supported GHC versions here: https://haskell-language-server.readthedocs.io/en/latest/supported-versions.html

Supporting new GHC versions is always a lot of work and often takes month before HLS can catch up :/

**Comment by fendor** [comment]
The error message has been improved and tells you now that the version is unsupported.
